src/publisher/api_v2_views.py

```diff
@@ -129,12 +128,16 @@ def article_fragment(request, art_id, fragment_id):
         reserved_keys = [XML2JSON]
         ensure(fragment_id not in reserved_keys, "that key is taken")
         with transaction.atomic():
+            article = models.Article.objects.get(manuscript_id=art_id)
+            data = request.data
+            frag, created, updated = fragment_logic.add(article, fragment_id, data, update=True)
```

Comment:
right so we only need to do this once as all `av` were pointing to the same article